File: 2808/2808.go
func minimumSecondsToEqualizeACircu(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := minimumSecondsToEqualizeACircu(root.Left)
	r := minimumSecondsToEqualizeACircu(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}

File: 2808/2808.cs
public class Solution {
    public int MinimumSecondsToEqualizeACircu(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(MinimumSecondsToEqualizeACircu(root.left), MinimumSecondsToEqualizeACircu(root.right));
    }
}